packages/components/psammead-topic-tags/src/index.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import {
+  string,
+  element,
+  object,
+  oneOf,
+  oneOfType,
+  shape,
+  arrayOf,
+} from 'prop-types';
+import styled from '@emotion/styled';
+import { C_LUNAR, C_EBON, C_METAL } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_HLF, GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { getBrevier } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+const [TAG_TEXT_PAD_Y, TAG_TEXT_PAD_X, ROW_SPACING] = [
+  '0.375rem',
+  '0.4375rem',
+  '0.0625rem',
+];
+
+const topicTagParent = `
```

Comment:
```suggestion
const topicTagStyles = `
```
Are these styles for an individual topic tag, just found the naming a bit confusing? If that is the case maybe `topicTagStyles`?

Reply:
Yeah good point that identifier is quite ambiguous, I'll make the change 👍